dclab/rtdc_dataset/check.py

```diff
@@ -97,7 +97,8 @@
     "online_contour": [
         # introduced in 0.34.0
         "bg empty",
-    ]
+    ],
+    "user": [],
```

Comment:
I doubt that this is enough.

Reply:
I reckon this is something I need an overview on (how these checks work)

Reply:
Just seeing the below comment, will look into that now.